src/slowlog.c

```diff
@@ -80,8 +87,8 @@ slowlogEntry *slowlogCreateEntry(client *c, robj **argv, int argc, long long dur
         }
     }
     se->time = time(NULL);
-    se->duration = duration;
-    se->id = server.slowlog_entry_id++;
+    se->cost = cost;
```

Comment:
Why not call this "size"?

Reply:
Because this is also used by slowlog, in which it represents the time duration.